content/docs/command-reference/exp/run.md

```diff
@@ -180,27 +192,36 @@ experiment we just ran (`exp-44136`).
 ## Example: Modify parameters on-the-fly
 
 `dvc exp run--set-param` (`-S`) saves you the need to manually edit the params
-file before running an experiment.
+file before running an experiment. It can override (`foo.bar=value`), append
+(`+foo.bar=value`), or remove (`~foo.bar`) parameters.
+
+<admon type="tip">
+
+DVC supports the
+[Hydra Override Grammar](https://hydra.cc/docs/advanced/override_grammar/basic/),
+with the exception of patterns that modify Hydra's defaults list.
+
+</admon>
+
+You can optionally provide a prefix `[<filename>:]` in order to edit a specific
+`dvc params` file. If not provided, `params.yaml` will be used as default.
 
 ```dvc
-$ dvc exp run -S prepare.split=0.25 -S featurize.max_features=2000
+$ dvc exp run -S custom_file.json:+newparam=foo
 ...
-Reproduced experiment(s): exp-18bf6
 ```
 
-To see the results, you can use `dvc exp diff`. It compares both params and
-metrics to the previous project version:
-
 ```dvc
-$ dvc exp diff
-Path         Metric    Value    Change
-scores.json  avg_prec  0.58187  -0.022184
-scores.json  roc_auc   0.93634  -0.024464
-
-Path         Param                   Value    Change
-params.yaml  featurize.max_features  2000     -1000
-params.yaml  prepare.split           0.25     0.05
+$ dvc params diff
+Path              Param      HEAD    workspace
+custom_file.json  new_param  -       foo
 ```
 
-> Notice that experiments run as a series don't build up on each other. They are
```

Comment:
Removed this note at the bottom of the page that felt a little out of context @jorgeorpinel . Not sure when/why it was added